addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -179,14 +185,7 @@ class ComposerTextInput extends Component {
      * @private
      */
     _onFocusinTextarea() {
-        this.composer.update({ hasFocus: true });
-    }
-
-    /**
-     * @private
-     */
-    _onFocusoutTextarea() {
```

Comment:
Why remove focusout management? On clicking anywhere else on the interface than on the thread, we would like the composer to be marked as not focused no? Is that handled somewhere else I'm missing?

Reply:
I think it's a mistake from me, as in my uses cases the composer was focused out by the new focus method